Uplift/Views/Cells/FacilitiesDropdownCollectionViewCell.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCollectionViewCell: UICollectionViewCell {
+    var collectionView: UICollectionView!
+    var collectionViewHeight: CGFloat!
+    var dropdownView: DropdownView!
+    var facility: Facility!
+    var headerImageView: UIImageView!
+    var headerNameLabel: UILabel!
+    var headerOpenLabel: UILabel!
+    var headerView: UIView!
+    var headerViewHeight: CGFloat = 52
+    var height: CGFloat!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupDropdownView()
+        setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configure(for facility: Facility) {
+        self.facility = facility
+        setupDropdownView()
+        setupConstraints()
+    }
+    
+    func setupDropdownView() {
+        headerView = setupHeaderView(headerString: facility.name.uppercased())
+        
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: StretchyHeaderLayout())
```

Comment:
why are we using `StretchyHeaderLayout`? isn't this the layout we use on the detail view controllers to get the stretch effect on the header images?